pkgs/top-level/python-packages.nix

```diff
@@ -8744,6 +8744,8 @@ in {
 
   sqlalchemy = callPackage ../development/python-modules/sqlalchemy { };
 
+  sqlalchemy_1_3 = callPackage ../development/python-modules/sqlalchemy/1.3.nix { };
```

Comment:
This is not allowed. Please move this into grab-site.

Reply:
Thanks, fixed and back to matching the other packages that require SQLAlchemy 1.3.